woocommerce/woocommerce-gutenberg-products-block
# Product Gallery Pager: Remove unnecessary margin

## What

Fixes #11285

## Why

This PR removes the excess **top** & **bottom** margins from the **Product Gallery Pager**.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. With a block theme, open `Appearance > Editor`.
2. Open `Templates > Single Product`.
3. Select `Convert to Blocks` if you are using the Classic Templates.
4. Add the `Product Gallery` block.
5. Ensure the **Pager** is enabled.
6. Verify that there's no Pager top or bottom margin both in the editor and frontend.


## Screenshots or screencast



**Editor:**

| Before | After |
| ------ | ----- |
|<img width="1091" alt="Single_Product_‹_Template_‹_Editor_‹_newproductgallery_—_WordPress" src="https://github.com/woocommerce/woocommerce-blocks/assets/905781/0ee137d3-a6bb-4b83-80f7-5b27e8e37c26">|<img width="1015" alt="Single_Product_‹_Template_‹_Editor_‹_newproductgallery_—_WordPress" src="https://github.com/woocommerce/woocommerce-blocks/assets/905781/5b851a4a-de03-4ca1-afc0-9a2a7ab93dbf">|

**Frontend:**

| Before | After |
| ------ | ----- |
|<img width="1161" alt="V-Neck_T-Shirt_–_newproductgallery" src="https://github.com/woocommerce/woocommerce-blocks/assets/905781/8a7ed41a-47c6-4135-90b3-e00a8b80de43">|<img width="1032" alt="V-Neck_T-Shirt_–_newproductgallery" src="https://github.com/woocommerce/woocommerce-blocks/assets/905781/e9ec92cc-b7a2-4f73-af04-d59a90e2a148">|


## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Product Gallery Pager: Remove unnecessary margin.
